backend/events.go

```diff
@@ -49,6 +58,21 @@ const (
 	Unknown                           //< Returned when the QueryContextCallback does not know how to deal with the given context.
 )
 
+// Add the InitCallback to the InitEvent to be called during initialization.
+// This should be called in a module's init() function.
+func (ie *InitEvent) Add(i InitCallback) {
+	*ie = append(*ie, i)
+}
+
+// Execute the InitEvent.
```

Comment:
Newb question. Shouldn't this godoc start with the name of the function?

`call Execute the InitEvent `